### PR TITLE
feat(sdk): update chain_id and contracts address Thanos Sepolia

### PR DESCRIPTION
**Pull request by 0x6e616d**

### CODE REVIEW

File: packages/tokamak/sdk/src/interfaces/types.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export enum L2ChainID {
   ZORA_MAINNET = 7777777,
   MODE_SEPOLIA = 919,
   MODE_MAINNET = 34443,
-  THANOS_SEPOLIA_TEST = 111551118080,
+  THANOS_SEPOLIA_TEST = 111551119090,
```
Comment (rlgns98kr) [inline packages/tokamak/sdk/src/interfaces/types.ts:40]: Can you delete the TEST? (THANOS_SEPOLIA_TEST -> THANOS_SEPOLIA)

File: packages/tokamak/sdk/src/utils/chain-constants.ts [hunk 2]
```diff
@@ -368,17 +368,17 @@ export const CONTRACT_ADDRESSES: {
   // internal testnet
   [L2ChainID.THANOS_SEPOLIA_TEST]: {
```
Comment (rlgns98kr) [inline packages/tokamak/sdk/src/utils/chain-constants.ts:369]: Can you delete the TEST? (THANOS_SEPOLIA_TEST -> THANOS_SEPOLIA)

File: packages/tokamak/sdk/src/interfaces/types.ts (hunk 1 above)
Comment (0x6e616d) [reply packages/tokamak/sdk/src/interfaces/types.ts]: Done

**Review by rlgns98kr — APPROVED** [review]
Thank you very much!

**Pull request merged**